tests/sync/test_syncing.py

```diff
@@ -7,32 +8,76 @@
 from kitovu import utils
 from kitovu.sync import syncing
 from kitovu.sync.plugin import smb
+from kitovu.sync.settings import ConnectionSettings
+from kitovu.sync.syncplugin import AbstractSyncPlugin
 from helpers import dummyplugin
 
 
+class FakePlugin(AbstractSyncPlugin):
```

Comment:
Can't we use the `DummyPlugin` here instead of having another similar thing?

Reply:
The reason was that I wanted to set the connection_schema, but I've now added an option to set it in the `DummyPlugin` constructor